libpod/sql_state_internal.go

```diff
@@ -15,6 +15,137 @@ import (
 	_ "github.com/mattn/go-sqlite3"
 )
 
+// Checks that the DB configuration matches the runtime's configuration
+func checkDB(db *sql.DB, r *Runtime) (err error) {
```

Comment:
The work that's going on in here for checkDB is not what I'd expect for a checkDB function.  I thought I'd find just a simple query with a validation of the value(s) like you have starting at line 104.  The rest of this code feel like it should be in a createRuntimeTable function to initially create the table, then another function to be called to insert/update values into it.